apps/L1E/instagram/instagram-frontend/src/app/(main)/(auth)/signin/page.tsx

```
@@ -0,0 +1,104 @@
+    })
+
+    const form = useForm<z.infer<typeof formSchema>>({
+        resolver: zodResolver(formSchema),
+        defaultValues: {
+            email: "",
+            password: "",
+        },
+    });
+
+    const onSubmit = async () => {  
+    };
```

ESLint

[error] Unexpected empty async arrow function.

(@typescript-eslint/no-empty-function)